community/lucene-index/src/test/java/org/neo4j/kernel/api/impl/index/LuceneSchemaIndexUniquenessVerificationIT.java

```diff
@@ -298,7 +298,7 @@ private void insert( Collection<PropertyValue> data ) throws IOException
         PropertyValue[] dataArray = data.toArray( new PropertyValue[data.size()] );
         for ( int i = 0; i < dataArray.length; i++ )
         {
-            Document doc = LuceneDocumentStructure.documentRepresentingProperty( i, dataArray[i].value );
+            Document doc = LuceneDocumentStructure.documentRepresentingProperties( (long) i, dataArray[i].value );
```

Comment:
Why this cast to `long`? It should automatically coerce. Similarly in `SimpleUniquenessVerifierTest`. Not a big issue, though.